neuroquery/tests/test_datasets.py

```diff
@@ -3,15 +3,15 @@
 from unittest import mock
 import pytest
 
+import pandas as pd
+
 from neuroquery import encoding
 from neuroquery import datasets
 
 
 class _FileResponse(object):
     def __init__(self, data_file):
-        self.data_file = str(
-            pathlib.Path(__file__).parent / "data" / data_file
-        )
+        self.data_file = str(pathlib.Path(__file__).parent / "data" / data_file)
```

Comment:
just for future PRs: in general it is good to avoid to make any changes that are not necessary for the specific purpose of your PR, because it adds distracting lines to the diff and makes the PR harder to review. so if your editor makes formatting changes to the whole file or something like that, you may want to check its configuration so that it only touches lines that you modified anyway. it's definitely not a big deal though so no need to revert it here; this PR is ready to be merged